test-lit/semantics/equivalence01.f90

```diff
@@ -1,3 +1,6 @@
+! RUN: not %flang -fdebug-resolve-names -fparse-only %s 2>&1 | FileCheck --check-prefixes=OK,ERROR %s
```

Comment:
!OK is probably a comment and not a prefix to check?

Reply:
Another prefix-like comment...I'll just have to add this one as a non-prefix.